server/src/test/java/org/elasticsearch/search/aggregations/bucket/histogram/DateHistogramAggregatorTests.java

```diff
@@ -50,7 +50,6 @@
 public class DateHistogramAggregatorTests extends AggregatorTestCase {
 
     private static final String DATE_FIELD = "date";
-    private static final String INSTANT_FIELD = "instant";
```

Comment:
Now that I look at the search index to run the agg I index the dates as well as make doc values for them. This makes this field not really useful any more.

Reply:
Good call! I think I removed those tests on accident. I'll have a go at adding theme back.